build-scripts/createRelease.ts

```diff
@@ -7,9 +7,18 @@ import * as child_process from 'child_process'
 import * as fs from 'fs-extra'
 import * as path from 'path'
 
+let releaseVersion: string = require(path.join('..', 'package.json')).version
+
+// Append SHA of commit if there are no tags
+let shaTag = ''
+if (child_process.execSync('git tag -l --contains HEAD').toString() === '') {
+    releaseVersion = releaseVersion.replace('-SNAPSHOT', '')
+    shaTag = child_process.execSync('git rev-parse --short=10 HEAD').toString()
+    releaseVersion += '-' + shaTag
```

Comment:
can avoid this line by swapping the order

    releaseVersion = releaseVersion.replace('-SNAPSHOT', `-${shaTag}`)